File: C - Progarm/Q9.c
//Question:
The following program intends to check if a number is even, but it contains an error.

#include <stdio.h>

int main() {
    int num = 4;
    if (num = 0) // Error: Should use == instead of =
        printf("Even number");
    else
        printf("Odd number");
    return 0;
}

//Expected Output
    Even number

File: Java-Program/Q9.java
//Question:
Fix the infinite loop.

public class InfiniteLoop {
    public static void main(String[] args) {
        int i = 0;
        while (i < 5) {
            System.out.println(i);
        }
    }
}

//Expected Output:
    0
    1
    2
    3
    4
